aioh2/helper.py

```diff
@@ -84,6 +84,6 @@ def factory():
 
 
 if hasattr(asyncio, 'ensure_future'):  # Python >= 3.5
-    async_task = asyncio.ensure_future
+    async_task = getattr(asyncio, 'ensure_future')
 else:
-    async_task = asyncio.async
+    async_task = getattr(asyncio, 'async')
```

Comment:
What are these for, muting IDE warnings?

Reply:
Python 3.7 (maybe even earlier versions) will through an exception with the previous code:

```
>>> if hasattr(asyncio, 'ensure_future'):  # Python >= 3.5
...     async_task = asyncio.ensure_future
... else:
...     async_task = asyncio.async
  File "<stdin>", line 4
    async_task = asyncio.async
                             ^
SyntaxError: invalid syntax
```
Most likely because `async()` is not available anymore. The `getattr()` calls should fix this behavior.